easybuild/easyblocks/i/icc.py

```diff
@@ -58,6 +60,15 @@ class EB_icc(IntelBase):
         - will fail for all older versions (due to newer silent installer)
     """
 
+    def configure_step(self):
+        """have fallback for components for > 2016 versions"""
+
+        if LooseVersion(self.version) >= LooseVersion('2016') and not self.cfg['components']:
+            self.log.warning("No components specified, falling back to ALL")
+            self.cfg.update('components', ['ALL'])
```

Comment:
or just not specify `components` at all if `components` easyconfig parameter is not defined